content/en/docs/concepts/storage/volumes.md

```diff
@@ -31,6 +31,8 @@ Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod"
 can use any number of volume types simultaneously.
 [Ephemeral volume](/docs/concepts/storage/ephemeral-volumes/) types have a lifetime of a pod, 
 but [persistent volumes](/docs/concepts/storage/persistent-volumes/) exist beyond
+can use any number of [volume types](#volume-types) simultaneously.
```

Comment:
Thanks! I appreciate the efforts of adding the link, which works as expected. 

However, I'm finding it difficult to understand the advantages of linking a section that immediately follows on the same page. Is there a specific advantage or purpose behind the decision to include a link to the next section, considering that a user would likely be reading it anyway?

Reply:
@dipesh-rawat  as per issue #40767  the needs  for change in the navigational link is mentioned ( "Get an idea about supported volume types ").

Reply:
@dipesh-rawat 
Even though it is next session, in the line itself readers can get to know volume types are listed below.